ubuntu_18.04-dev.dockerfile

```diff
@@ -44,7 +53,7 @@ ENV LIBRARY_PATH $HOME/opt/moab/lib:$LIBRARY_PATH
 
 # Install PyNE
 RUN cd $HOME/opt \
-    && git clone https://github.com/pyne/pyne.git \
+    && git clone https://github.com/cnerg/pyne.git \
```

Comment:
Since these are the install scripts distributed to install the PyNE develop, I think this should stay as `pyne` not `cnerg`.